examples/xdg_shell_v6_test.rs

```diff
@@ -75,17 +94,7 @@ impl XdgV6ShellManagerHandler for XdgV6ShellManager {
                 }
             }).expect("Layout was destroyed");
         }).unwrap();
-        Some(Box::new(XdgV6ShellHandlerEx))
-    }
-
-    fn surface_destroyed(&mut self, compositor: CompositorHandle, shell: XdgV6ShellSurfaceHandle) {
-        with_handles!([(compositor: {compositor})] => {
-            let state: &mut State = compositor.into();
-            let weak = shell;
-            if let Some(index) = state.shells.iter().position(|s| *s == weak) {
-                state.shells.remove(index);
-            }
-        }).unwrap();
+        Some((Box::new(XdgV6ShellHandlerEx), Box::new(SurfaceEx)))
```

Comment:
Instead of Option of tuple, how about tuple of Option?

Reply:
Fixed in 123517b8